AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -339,6 +340,15 @@ extension DefaultVideoClientController: VideoClientDelegate {
             }
         }
     }
+    
+    public func videoClientTurnURIsReceived(_ uris: [String]?) -> [String]? {
```

Comment:
I didn't see this in [`VideoClientDelegate`](https://code.amazon.com/packages/ChimeTincan/blobs/b9c4862a9e845366cb23186033fa52bb9db71f94/--/media_client/apple/MediaClient/VideoClient/VideoClient/VideoClientSDK.h#L125). Am I missing something?

Reply:
It is part of the associated media layer change.